### PR TITLE
add tests for notification models

### LINKED ISSUE #7: [3pts] Setup Notification Model & Controller & Tests
- Create Notification Models
- Create Notification Controllers
- Add Notification Ressource tests

### PR DESCRIPTION
**Pull request by LYANGEND**
**In this pull request I have done the following**;

- Add tests for the notification models

closes https://github.com/ichala/short-let-api/issues/7

### CODE REVIEW

File: spec/requests/notification_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Notifications', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```
Comment (ichala) [inline spec/requests/notification_spec.rb]: @LYANGEND kindly remove this file , its not required for now

**Review by ichala — CHANGES_REQUESTED** [review]
🛑 Required Changes 🛑

File: spec/requests/notification_spec.rb (hunk 1 above)
Comment (LYANGEND) [reply spec/requests/notification_spec.rb]: Alright 👍

**Comment by ichala** [comment]
Approved for me ✅

**Review by EidHachem — APPROVED** [review]
Great Job @LYANGEND 

# Approved ✔️ 

It's time to merge 🎉

**Review by ichala — APPROVED** [review]

**Review by d2ndjim — APPROVED** [review]
@LYANGEND, great job implementing the test, all tests are passing
### APPROVED ✔️ ✔️ ✔️

**Review by aimalamiri — APPROVED** [review]
Hi @LYANGEND, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**